application/src/objects/obstacles/abstract-dynamic-obstacle.js

```diff
@@ -31,9 +31,13 @@ AbstractDynamicObstacle.prototype.getConstants = function()
     };
 };
 
-AbstractDynamicObstacle.prototype.createPhysicsBody = function(state)
+AbstractDynamicObstacle.prototype.createPhysicsBody = function(state, rotation)
 {
     state.game.physics.p2.enable(this);
+
+    if (rotation) {
+        this.body.rotation = rotation * Math.PI / 180;
```

Comment:
I had a degreesToRadians method that I added to util.js but I didn't commit it because I didn't end up using it. Maybe we could use it after all.